doc/source/index.rst

```diff
@@ -37,26 +42,25 @@ Those two tasks are carried out by two classes:
    in a model comparison or scenario exercise following the IAMC data format.
 
 #. The :class:`RegionProcessor` class carries out renaming and aggregation based on
-   information given in model mapping files.
+   information given in yaml model mapping files.
 
-A starting point on how to use these two classes with existing data templates and model mappings can be found in :ref:`getting-started`.
+Instructions on how to install and get started using nomenclature can be found in
```

Comment:
Is it really necessary to add explicit references to these sections when the TOC is shown just below?

Reply:
Removed the references.